utils/dockerutil.py

```diff
@@ -134,8 +135,12 @@ def client(self):
             self._client.ping()
             return self._client
         except Exception as ex:
-            log.error("Failed to initialize the docker client. Docker-related features "
-                "will fail. Will retry %s time(s). Error: %s" % (self.left_init_retries, str(ex)))
+            # No docker_daemon.yaml + no existing Docker socket file = Dockerless
+            if self._no_conf_file and not os.path.isfile(DEFAULT_SOCK_PATH):
+                log.info("No docker_daemon conf and no Docker instance detected.")
```

Comment:
We could but, without the docker_daemon conf, it will always retry 0 times right? What if we go ahead and say that Docker features have been disabled.
`"No docker_daemon conf and no Docker instance detected. Docker features disabled."`